QuizApp/Quiz/View/QuizViewModel.swift

```diff
@@ -3,6 +3,8 @@ import UIKit
 class QuizViewModel {
 
     @Published var categories: [SelectableCategoryModel] = []
+    @Published var isQuizzesExist: Bool = true
+    @Published var isDataReached: Bool = true
```

Comment:
```suggestion
    @Published var quizzesFetchFailed: Bool = false
```
you don't have to necessarily use these names, but just something that is more explanatory